catalog-items.html

```diff
@@ -29,24 +29,56 @@
                 <li class="nav-item"> 
                     <a href="index.html" class="nav-dropbtn" >Home</a>
 
-                    <ul class="nav-dropdown-content">
-                        <a href="catalog.html" class="nav-link">Catalog</a>
-                        <a href="contact.html" class="nav-link">Contact us</a>
+                    <ul>
+                        <li><a href="#"></a>#</li>
+                        <li><a href="#"></a>#</li>
                     </ul>
                 </li>
 
                 <li class="nav-item"> 
-                    <a href="index.html" class="nav-dropbtn" >Home</a>
+                    <a href="about.html" class="nav-dropbtn" >About</a>
+
+                    <ul>
+                        <li><a href="#"></a>#</li>
+                        <li><a href="#"></a>#</li>
+                    </ul>
+                </li>
+
+                <li class="nav-item"> 
+                    <a href="roadmap.html" class="nav-dropbtn" >Roadmap</a>
```

Comment:
Small not really important code style notice: will be better to close brackets `>` identically everywhere: without a space.
```suggestion
                    <a href="roadmap.html" class="nav-dropbtn">Roadmap</a>
```

Reply:
appreciate for correction kind of this thing
I really want to write clear code